src/arrays.jl

```diff
@@ -441,6 +459,13 @@ for op in (:(Base.:*), :(Base.:/), :(Base.:\))
     end
 end
 
+# Cover method ambiguities from, e.g., op(::Array, ::Quantity)::QuantityArray`
```

Comment:
Disambiguities should go in src/disambiguities

Reply:
To be more specific: the new “ Eagerly construct a `QuantityArray` from the following:” part can stay here. But specific disambiguities should go in the disambiguities file